spec/views/catalog/_index_list_default.html.erb_spec.rb

```diff
@@ -21,13 +22,15 @@
     allow(User).to receive(:find_by_user_key).and_return(joe, justin)
     allow(view).to receive(:blacklight_config).and_return(CatalogController.blacklight_config)
     allow(view).to receive(:blacklight_configuration_context).and_return(blacklight_configuration_context)
+    allow(view).to receive(:search_state).and_return(search_state)
+    allow(view).to receive(:search_action_path)
     render 'catalog/index_list_default', document: document
   end
 
   it "displays metadata" do
     expect(rendered).not_to include 'Title:'
     expect(rendered).to include '<span class="attribute-label h4">Creator:</span>'
-    expect(rendered).to include '<span itemprop="creator">Justin</span> and <span itemprop="creator">Joe</span>'
+    expect(rendered).to include '<span itemprop="creator"><a href="/catalog"><span itemprop="creator">Justin</span></a></span> and <span itemprop="creator"><a href="/catalog"><span itemprop="creator">Joe</span></a></span>'
```

Comment:
It probably doesn't have those parameters because `search_state` is being stubbed to return a value that has no parameters set.